api/src/main/java/org/openmrs/CohortMembership.java

```diff
@@ -142,6 +141,6 @@ public int compareTo(CohortMembership o) {
 		if (ret == 0) {
 			ret = this.getUuid().compareTo(o.getUuid());
 		}
-		return ret;
+		return set;
```

Comment:
Thanks for taking time to work on this, just a few comments;
1: Did you see the travis failure?
2: Are you even sure this file compiles?
3: Is this all the ticket required?

Reply:
> Thanks for taking time to work on this, just a few comments;
> 1: Did you see the travis failure?
> 2: Are you even sure this file compiles?
> 3: Is this all the ticket required?

i saw the travis failure

Reply:
Can you fix it?

Reply:
> Are you sure that you followed all the instructions there?

is there anything that i could have missed?

Reply:
Can you take a second look at you confirm that you did all that is mentioned on that pull request tips page?